src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java

```diff
@@ -221,7 +221,10 @@ public void chunkLimits(BlockPlaceEvent event) {
 				if (state != null && mat.equals(state.getType())) {
 					if (++current > limit) {
 						event.setCancelled(true);
-						player.sendMessage(config.getChunkLimitsExceededMessage());
+						player.sendMessage(config.getChunkLimitsExceededMessage()
+								.replaceAll("%Limit%",Integer.toString(limit))
```

Comment:
Could you make these either all upper case (preferred) or all lower case? Capitalization like that usually isn't used in this context.

Reply:
I tried to follow the format set by other SAH config replacements

- [message: '%Victim% was combat tagged by %Attacker%'](https://github.com/CivClassic/AnsibleSetup/blob/fde59ac5a2c08470a3dae847544ee3168a986aff/templates/public/plugins/SimpleAdminHacks/config.yml.j2#L56)
- [announce: '&f%Player% is brand new!'](https://github.com/CivClassic/AnsibleSetup/blob/fde59ac5a2c08470a3dae847544ee3168a986aff/templates/public/plugins/SimpleAdminHacks/config.yml.j2#L60)

I can change it to all uppercase, should I edit all of SAH config replacements to be uppercase?

Reply:
No, just follow the convention then I guess. Weird.